src/index.ts

```diff
@@ -25,6 +25,8 @@ const main = async (): Promise<void> => {
   switch (context.eventName) {
     case 'check_suite':
       return checkSuiteHandle(octokit, GITHUB_LOGIN, MAXIMUM_RETRIES);
+    case 'workflow_run':
+      return checkSuiteHandle(octokit, GITHUB_LOGIN, MAXIMUM_RETRIES);
```

Comment:
This method name `checkSuiteHandle` implies that it will handle the `check_suite` type.

Maybe better to call this `workflowRunHandle` and have a dedicated file (`src/eventHandlers/workflowRun/index.ts` for it?

Reply:
You prefer that I duplicate this file completely?

Reply:
Not perse. 

If the code will be duplicated (which I think will be) then we could encapsulate that in a more generic function `defaultHandle` and make the `checkSuiteHandle` and the `workflowRunHandle` reach out to `defaultHandle`

Reply:
Finally, I use `continuousIntegrationEndHandle` does it sound good?

Reply:
Now CI pass again :-)